slurmci-test.toml

```
@@ -58,6 +58,9 @@ gpu = [
   { file = "tutorials/Atmos/dry_rayleigh_benard.jl", slurmargs = ["--ntasks=1"], args = ["--fix-rng-seed"] },
   { file = "tutorials/Atmos/risingbubble.jl", slurmargs = ["--ntasks=3"], args = [] },
   { file = "experiments/AtmosGCM/heldsuarez.jl", slurmargs = ["--ntasks=1"], args = ["--diagnostics=default", "--fix-rng-seed"] },
+  { file = "experiments/AtmosGCM/baroclinic-wave.jl", slurmargs = ["--ntasks=1"], args = ["--diagnostics=default"] },
+  { file = "experiments/AtmosGCM/nonhydrostatic-gravity-wave.jl", slurmargs = ["--ntasks=1"], args = ["--diagnostics=default"] },
+  { file = "experiments/AtmosGCM/isothermal-zonal-flow.jl", slurmargs = ["--ntasks=1"], args = ["--diagnostics=default"] },
```

```suggestion
  { file = "experiments/AtmosGCM/baroclinic-wave.jl", slurmargs = ["--ntasks=1"], args = [] },
  { file = "experiments/AtmosGCM/nonhydrostatic-gravity-wave.jl", slurmargs = ["--ntasks=1"], args = [] },
  { file = "experiments/AtmosGCM/isothermal-zonal-flow.jl", slurmargs = ["--ntasks=1"], args = [] },
```
Diagnostics are enabled (with `--diagnostics=default`) in Held-Suarez to test the GCM diagnostics (the interval was chosen such that diagnostics output would happen only once). There's no need to test the same code path more than once.